features/cart/viewing_cart/adding_product_with_options_to_cart.feature

```diff
@@ -0,0 +1,19 @@
+@viewing_cart
+Feature: Adding a product with selected option to the cart
+    In order to select products for purchase
+    As a Visitor
+    I want to be able to add products with selected options to cart
+
+    Background:
+        Given the store operates on a single channel in "France"
+
+    @ui
+    Scenario: Adding a product with multiple options to the cart
+        Given the store has a product "T-shirt banana"
+        And this product has option "Size" with value "S" priced at "€12.35"
```

Comment:
The variant name is optional, so we can modify the scenario:
`And this product is available in "S" size`